Updating the Firmware of your Skywallet.md

```diff
@@ -0,0 +1,61 @@
+# Updating the Firmware of your Skywallet
+
+The firmware of your Skywallet is its user interface, the software code that interacts with the hardware. And like any other software, the firmware of your Skywallet also needs periodic updations.
```

Comment:
I would first mention, that the firmware needs to be installed when the Skywallet comes shipped first time. Also Updations is not a word afaik.

Reply:
Maybe we do not need to explain what firmware is. I would just say that it comes without firmware initially and that it should be updated regularly.